### PR TITLE
Remove paginate

### PR DESCRIPTION
**Pull request by MarkusGnigler**
I removed the obsolete Paginate method from issue [Remove obsolete Paginate builder action.](https://github.com/ardalis/Specification/issues/189).

Hope this change was sufficient.

### CODE REVIEW

**Comment by fiseni** [comment]
Hey @MarkusGnigler,

Thank you for your help. In the issue, I also mentioned the `IsPagingEnabled` property. Please can you remove that too? It's no longer necessary.

File: Specification/src/Ardalis.Specification/Exceptions/DuplicateSkipException.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ namespace Ardalis.Specification
 {
     public class DuplicateSkipException : Exception
     {
-        private const string message = "Duplicate use of the Skip(). Ensure you don't use both Paginate() and Skip() in the same specification!";
+        private const string message = "Duplicate use of the Skip(). Ensure you don't use Skip() in the same specification!";
```
Comment (fiseni) [inline Specification/src/Ardalis.Specification/Exceptions/DuplicateSkipException.cs:9]: Perhaps the message should be "Duplicate use of Skip(). Ensure you don't use Skip() more than once in the same specification!".

File: Specification/src/Ardalis.Specification/Exceptions/DuplicateTakeException.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ namespace Ardalis.Specification
 {
     public class DuplicateTakeException : Exception
     {
-        private const string message = "Duplicate use of Take(). Ensure you don't use both Paginate() and Take() in the same specification!";
+        private const string message = "Duplicate use of Take(). Ensure you don't use Take() in the same specification!";
```
Comment (fiseni) [inline Specification/src/Ardalis.Specification/Exceptions/DuplicateTakeException.cs:9]: Same as for Skip()

**Review by fiseni — CHANGES_REQUESTED** [review]

**Comment by MarkusGnigler** [comment]
Hy @fiseni,

oh sure, it's a better exception message. Thanks for your guidance.

I also remove the `IsPagingEnabled` prop, sorry for the overlooking.

**Comment by MarkusGnigler** [comment]
Should I also refactor the skip and take builder tests to use more the FluentAssertion library, or is using classic assertions the preferred approach since it's shorter?

Tests are located in:
- Ardalis.Specification.UnitTests.SpecificationBuilderExtensions_Skip
- Ardalis.Specification.UnitTests.SpecificationBuilderExtensions_Take

```cs
[Fact]
public void ThrowsDuplicateSkipException_GivenSkipUsedMoreThanOnce()
{
    Assert.Throws<DuplicateSkipException>(() => new StoreDuplicateSkipSpec());
}
```

```cs
[Fact]
public void ThrowsDuplicateSkipException_GivenSkipUsedMoreThanOnce()
{
    Action sutAction = () => new StoreDuplicateSkipSpec();

    sutAction.Should()
        .Throw<DuplicateSkipException>();
}
```

**Review by fiseni — APPROVED** [review]

**Comment by fiseni** [comment]
I don't mind really. Since we're using fluent assertions in all other places, you can refactor it if you want, so we're consistent.

**Comment by MarkusGnigler** [comment]
The consistent aspect was my first intention.

Should i scan other plaeses as well?

By me it's already evening, I'll try to commit in the next few days.

Wish you a happy new year

**Comment by fiseni** [comment]
No worries, take your time. Yes, you can refactor other places too. Thank you for the help, it's appreciated.

Happy New Year!

**Comment by fiseni** [comment]
Hey @MarkusGnigler. You requested a review but I don't see any new commits. Do you want me to merge it as it is, or do you want to work on the tests?

**Comment by MarkusGnigler** [comment]
Hi @fiseni.

Sry, i think the requested review was a mistake.

I don't know, should i create a new PR for the test refactoring or should i stick to the existing one?

**Comment by MarkusGnigler** [comment]
Hi @fiseni 

I've decided to commit the test assertions to this pr as well, since it's not that big impact.

I hope it's ok for you?

**Review by fiseni — APPROVED** [review]

**Comment by fiseni** [comment]
Thanks for the help!

**Comment by MarkusGnigler** [comment]
Thank you as well for your friendly communication and guidance!

**Pull request merged**